Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Collections/Editor/BaseCollectionEditor.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using UnityEditor;
+
+
+namespace Microsoft.MixedReality.Toolkit.SDK.UX.Collections
```

Comment:
`namespace Microsoft.MixedReality.Toolkit.SDK.UX.Inspectors.Collections`

Also, please update the editor folder name to be `Inspectors` instead.

Reply:
Folder location should also reflect the namespace I provided above.